src/software/embedded/thunderloop.cpp

```diff
@@ -86,7 +86,8 @@ Thunderloop::Thunderloop(const RobotConstants_t& robot_constants, bool enable_lo
       chip_pulse_width_(
           std::stoi(redis_client_->getSync(ROBOT_CHIP_PULSE_WIDTH_REDIS_KEY))),
       primitive_executor_(Duration::fromSeconds(1.0 / loop_hz), robot_constants,
-                          TeamColour::YELLOW, robot_id_)
+                          TeamColour::YELLOW, robot_id_),
+      robot_localizer_(0.5, 0.01 * 0.01, 0.5 * 0.5, 0.1 * 0.1)
```

Comment:
set these numbers up in `robot_constants.h` and pass them in (otherwise we have to keep looking in the constructor)

`PROCESS_NOISE_VARIANCE`
`TARGET_ANGULAR_ACCELERATION_NOISE_VARIANCE`
etc.